cpp/src/io/json/json_column.cu

```diff
@@ -689,19 +689,24 @@ std::pair<std::unique_ptr<column>, std::vector<column_name_info>> device_json_co
       size_type num_rows = json_col.child_offsets.size() - 1;
       std::vector<column_name_info> column_names{};
       column_names.emplace_back("offsets");
-      column_names.emplace_back(json_col.child_columns.begin()->first);
+      column_names.emplace_back(
+        json_col.child_columns.empty() ? "element" : json_col.child_columns.begin()->first);
```

Comment:
I'm guessing this name is only ever seen internally, so it probably doesn't matter much, but I would still prefer using some extremely obvious placeholder name.

Reply:
moving `list_child_name` as global constexpr.